numpy/core/_add_newdocs.py

```diff
@@ -4356,9 +4359,8 @@ def luf(lamdaexpr, *args, **kwargs):
     array([[1, 2],
            [4, 5]], dtype=int16)
     >>> y.view(dtype=[('width', np.int16), ('length', np.int16)])
-    Traceback (most recent call last):
-      File "<stdin>", line 1, in <module>
-    ValueError: new type not compatible with array.
```

Comment:
Showing the message is useful here

Reply:
Shouldn't be any cases left where I've obliterated Tracebacks anymore -- if there are, that's probably because they don't happen in execution anymore for whatever reason.

Reply:
Double checked the source -- the modernized traceback is in place here now so marking as resolved.